packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -646,6 +646,8 @@
 	"settings.communityNodes.viewDocsAction.label": "Documentation",
 	"settings.communityNodes.uninstallAction.label": "Uninstall package",
 	"settings.communityNodes.upToDate.tooltip": "You are up to date",
+	"settings.communityNodes.fetchError.title": "Error loading installed packages",
+	"settings.communityNodes.fetchError.message": "Please try again or contact our support",
```

Comment:
def don't include `contact our support`.. Simply remove description here, it's not needed.

Reply:
Got updated values from Max so will add those.